gbpservice/nfp/configurator/drivers/vpn/vyos/vyos_vpn_driver.py

```diff
@@ -745,7 +745,8 @@ def __init__(self, conf):
         self.conf = conf
         self.handlers = {
             'vpn_service': {
-                'create': self.create_vpn_service},
+                'create': self.create_vpn_service,
+                'delete': self.delete_vpn_service},
```

Comment:
Let us remove the delete the vpn service delete related calls as they are not needed for GBP case.